third_party/move/move-prover/tests/sources/regression/vector_theory_boogie_array.v2_exp

```diff
@@ -13,7 +13,7 @@ error: post-condition does not hold
    =         addr = <redacted>
    =     at tests/sources/regression/vector_theory_boogie_array.move:7: f1
    =     at tests/sources/regression/vector_theory_boogie_array.move:8: f1
-   =         pool = <redacted>
+   =         idx = <redacted>
```

Comment:
I don't understand this baseline change - I am assuming it has been vetted.

Reply:
This is an issue from the prover when generating the trace, which depends on the boogie output as far as I remember.